fwutil/lib.py

```diff
@@ -903,6 +903,13 @@ def is_first_auto_update(self, boot):
                 return False
         for status_file in glob.glob(os.path.join(FIRMWARE_AU_STATUS_DIR, FW_AU_STATUS_FILE)):
             if status_file is not None:
+                data = self.read_au_status_file_if_exists(FW_AU_STATUS_FILE_PATH)
+                if data is not None:
+                    boot_type = list(data.keys())[0]
+                    if boot_type is "none" and boot_type is not boot:
```

Comment:
Why `boot_type is not boot`?

This seems intended to prevent a user from consecutively running a "none" boot but I believe that should be possible.